### PR TITLE
Validate email

### PR DESCRIPTION
**Pull request by AlejandraKeber**
In this Milestone we are implementing a client-side validation of the data in the contact form before submission:
- Add js code to validate if the email is written in lower case and send a message to the submitter

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,10 +118,11 @@ <h2 class="contactme-title">Contact me</h2>
                 <p class="contactme-detail">If you have an application you are interested in developing, a feature that
                     you need built or a
                     project that needs coding. I’d love to help with it</p>
-                <form action="https://formspree.io/f/mzbwzepn" method="post">
+                <form id="signup" action="https://formspree.io/f/mzbwzepn" method="post">
                     <fieldset class="finalforms">
                         <input name="name" type="text" maxlength="30" placeholder="Alejandra" required>
-                        <input name="email" type="email" placeholder="alejandrakeber@gmail.com" required>
+                        <input id="email" name="email" type="email" placeholder="alejandrakeber@gmail.com" required>
+                        <p id="error"></p>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Good job so far 👏 
- [ ] as the stated in the requirements. 
- [ ] If the validation is not OK, you show an error message to the user `near the submit button` informing them of the error and the form is not sent.
- [ ] kindly move the error message near the submit button. 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @AlejandraKeber ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -164,3 +165,15 @@ for (let index = 0; index < openPopup.length; index += 1) {
 closePopup.addEventListener('click', () => {
   modalPopup.style.display = 'none';
 });
+
+/* Validate email form */
+const email = document.getElementById('email').value;
+const lowerCase = email.toLowerCase;
+const submit = document.querySelector('.submit');
+
+submit.addEventListener('click', (e) => {
```
Comment (Ismailco) [inline script.js]: - [ ] Even if the email is in lowercase the error message still shows and the form doesn't get submitted, kindly fix this so the message show only when the email is not in lowercase
- [ ] As the previous reviewer mentioned, kindly consider moving the error message close to the submit button

Screenshot_20220909_181311

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @AlejandraKeber, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @AlejandraKeber 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**